### DISCUSSION TITLE
Preview generator and Area blocks unable use in skeleton project in my machine

### DISCUSSION THREAD

**Discussion by arulselvan008** [pimcore/pimcore#13730] [Q&A / Support] [answered]
Hi, 

1.I downloaded the skeleton project but am unable to preview the data object class in the UI screen. (Error: preview not available, it seems that there's a problem with this object.)

2.On my screen, I am unable to add the + buttons to add the area blocks to the CMS page.

Please assist me in resolving the issues on my machine.

**Comment by mattamon** [comment, marked as answer]
Hey @arulselvan008,

which Pimcore version?

1. could you please share a screenshot of the error itself and which object you are trying to preview? 

2. How did you set up the area blocks in the code? Could you please share your approach?

**Reply by arulselvan008** [reply]
Hi ,

Am installed a pimcore skeleton in my ubuntu machine .

screen short shared.

![image](https://user-images.githubusercontent.com/106297450/206157679-a3182d86-dd7d-4d87-bea2-108a8020f936.png)

![image](https://user-images.githubusercontent.com/106297450/206156900-c1bd317a-a649-4377-ba4a-784202a449dc.png)

am referred the Link Generator document in the pimcore website .

**Reply by arulselvan008** [reply]
Version: v10.5.11

**Comment by arulselvan008** [comment]
Unexempt The server occasionally got an error, but we didn't change anything in the backend and restart the Apache and MySQL servers; the error is gone and the UI screen is working fine.

![MicrosoftTeams-image (5)](https://user-images.githubusercontent.com/106297450/206159331-f42fabe8-66e4-43a4-9eed-b73840fc3034.png)
![MicrosoftTeams-image (4)](https://user-images.githubusercontent.com/106297450/206159339-f11ff93c-0b54-459b-b91c-e91d3e717f6a.png)
![MicrosoftTeams-image (3)](https://user-images.githubusercontent.com/106297450/206159341-f48fbaa3-f5db-4839-801a-4a6b4ceb60f5.png)
